src/components/QueryBuilder.vue

```diff
@@ -23,6 +23,15 @@
 		<div class="querybuilder__run">
 			<Button @click.native="runQuery" type="primaryProgressive">Run query</Button>
 		</div>
+		<div class="querybuilder__result">
+			<div v-if="!showQueryResults" class="querybuilder__result__description">Results will be displayed here</div>
+			<iframe v-bind:src="'https://query.wikidata.org/embed.html#' + encodedQuery"
```

Comment:
I assume this would be accessible to third party installations of Wikibase as well, so this should be configurable. You can file a bug or todo somewhere for this.

Reply:
did you mean this: https://query.wikidata.org/embed.html should be configurable ?

Reply:
Yes

Reply:
i can try and set up and application config object containing Vue application global configurations.
or were you thinking somewhere in the direction of enviroment variables ?

Reply:
I was more thinking of the former (Vue global application configurations). If it grows too big, I'm fine with hard-coding it for now and doing it later (just tracking the tech debt somewhere, either a todo or a phabricator ticket)